### ISSUE TITLE
[Feature Request] Animation request for elements with VWindow like animation

### ISSUE THREAD

**Issue by Skrayvee** [vuetifyjs/vuetify#15542]
### Problem to solve
This animation will allow the application to be more native, and it will also be much cooler
Now it only works after you have released the element

### Proposed solution
Smooth animation when the window follows the finger.
At the moment, the animation is played only after that the finger was taken away from the screen.
The new animation should affect from Windows to VCarousel

**Comment by KaelWD** [comment]
Duplicate of #8612
